matter_server/README.md

```diff
@@ -13,3 +13,13 @@ integration communicates via WebSocket with this server.
 
 [aarch64-shield]: https://img.shields.io/badge/aarch64-yes-green.svg
 [amd64-shield]: https://img.shields.io/badge/amd64-yes-green.svg
+
+## Matter Server Update Procedure
```

Comment:
This will show in the add-on info page. Maybe we should move it to a separate markdown file, less visible for users?

Reply:
@frenck what would be your recommendation for how to document this? Move to eg `RELEASE.md` or keep it in the README.md, or something else?

Reply:
Well it is user-facing, so I like your suggestion for a `RELEASE.md` 👍